<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSaveParameters</class>
 <widget class="QDialog" name="DialogSaveParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сохранение параметров</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="parameterSaveAsLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="saveAsCheckBox">
     <property name="text">
      <string>Сохранить как</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="parameterSetLabel">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Unf</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>saveAsCheckBox</tabstop>
  <tabstop>parameterSaveAsLineEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveParameters</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveParameters</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
